webrender/src/render_backend.rs

```diff
@@ -1028,9 +1029,11 @@ impl RenderBackend {
             self.result_tx.send(msg).unwrap();
         }
 
-        for evt in notifications {
-            self.notifier.external_event(evt)
-        }
+        drain_filter(
+            &mut notifications,
+            |n| { n.when() == CheckPoint::AfterFrameBuilding },
```

Comment:
nit: `CheckPoint` should be [a single word](https://en.wikipedia.org/wiki/Checkpoint)